forest/src/cli/mod.rs

```diff
@@ -224,6 +217,31 @@ impl CliOpts {
     }
 }
 
+pub fn find_default_config() -> Option<Config> {
+    // check forest config path first, then look at other default locations
```

Comment:
From a user perspective, it may be nice to check all those places and display a piece of information if e.g. `forest` will use config from `XDG_CONFIG_HOME` and not what is there in `HOME_DIR`. Otherwise, we could at least document it somewhere in README.

Reply:
Not sure how others feel about it @lemmih @elmattic

Reply:
Yes, I think it would help to have some `info` log that displays to the user the loaded config file location.

Reply:
And in case of doubt the user still can add `config dump` to command-line to see the computed config.

Reply:
Added logging with config locations and errors that occured while trying to load the config. Resorts to default on error